app/src/main/res/layout-land/administrator_controls_download_permissions_view.xml

```diff
@@ -34,6 +34,7 @@
       android:layout_width="match_parent"
       android:layout_height="wrap_content"
       android:background="@color/white"
+      android:focusable="true"
```

Comment:
I'm still seeing a focus issue with TalkBack. Selecting/deselecting the switch is causing all focus to be lost. Do we know why?